internal/cmd/group_locations.go

```diff
@@ -155,3 +156,16 @@ var groupsLocationsRmCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func locationsCmdsArgs(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	client, err := createTursoClientFromAccessToken(false)
+	if err != nil {
+		return nil, cobra.ShellCompDirectiveNoFileComp | cobra.ShellCompDirectiveNoSpace
+	}
+	if len(args) == 0 {
+		// TODO: add completion for group names
+		return nil, cobra.ShellCompDirectiveNoFileComp | cobra.ShellCompDirectiveNoSpace
+	}
+	locations, _ := locations(client)
```

Comment:
Nit: We should probably filter out locations already present in args.

Reply:
Agree! Also, filter out locations already present in the DB itself.
But I see those as general improvements to our location autocompletion.
Let's address it separately.